mriqc/workflows/functional/base.py

```diff
@@ -72,21 +73,29 @@ def fmri_qc_workflow(name="funcMRIQC"):
     mem_gb = config.workflow.biggest_file_gb
 
     dataset = config.workflow.inputs.get("bold", [])
+    full_files = [
+        file for file in dataset if nb.load(file).shape[3] >= 5
+    ]
```

Comment:
What do you think about doing the best we can?

```suggestion
    full_files = []
    for bold_path in dataset:
        try:
            bold_len = nb.load(bold_path).shape[3]
        except nb.filebasedimages.ImageFileError:
            bold_len = config.workflow.min_len_bold
        except IndexError:    # shape has only 3 elements
            bold_len = 0

        if bold_len >= config.workflow.min_len_bold:
            full_files.append(bold_path)
        else:
            config.loggers.workflow.warn(
                f'Dismissing {bold_path} for processing: insufficient number of '
                f'timepoints ({bold_len}) to execute the workflow.'
            )
```

This will require the definition of a `config.workflow.min_len_bold` in the config module.

For users with datalad datasets with these short runs who have not done `get()` on the image, this will fail the first time they run it and work the second. WDYT?